public/scss/modules/_company.scss

```diff
@@ -29,8 +29,17 @@ Company
 
 .company-grid {
     .company-grid-item {
-        //border: 1px solid darken(#fff, 20%);
-        height: 25em;
+        border: 1px solid darken(#fff, 20%);
+        border-radius: 5px;
+        height: 17em;
+        margin-bottom: 8em;
```

Comment:
I think this might look better if it is 2em, since then the vertical spacing is equal to the horizontal spacing.